src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1012,7 +1021,11 @@ private function getProcess($commandline, $cwd = null, array $env = null, $stdin
             }
         }
 
-        return $process;
+        if (self::$process) {
+            self::$process->stop(0);
+        }
```

Comment:
because if won't force-stop the process if getProcess is called two times in the same test case